[IMP] Fleet: Update services (#13268)
odoo/documentation: content/applications/hr/fleet/service.rst

needed. Scheduling repairs and managing services for an entire fleet is necessary to ensure all
vehicles are in good working order when they are needed.

> Services can be logged in advance, such as regular maintenance, like oil changes or tire

Reviewer: ```suggestion
Services, such as regular maintenance, like oil changes or tire, can be logged in advance.
```

b/c those are examples of services in general, not services logged in advance specifically.

Fill out the information on the form. The only two fields that are required to be populated are
:guilabel:`Service Type` and :guilabel:`Vehicle`.

The fields on the form are:

- :guilabel:`Description`: Enter a brief description of the service in this field.

> - :guilabel:`Service Type`: Using the drop-down menu, select the type of service performed. Or, enter

Reviewer: Consider a conjunction besides 'or', especially since using it later in the sentence.

https://www.grammarly.com/blog/sentences/starting-a-sentence-with-a-conjunction/

  edit...` to :ref:`add the service type and configure it <fleet/new-type>`.

  .. important::
     Only one :guilabel:`Service Type` comes preconfigured in Odoo: :guilabel:`Vendor Bill`.

- :guilabel:`Date`: Using the calendar selector, select the date the service was provided, or
  is scheduled to be performed. Navigate to the desired month using the :icon:`oi-chevron-left`
  :icon:`oi-chevron-right` :guilabel:`(arrow)` icons, then click on the date to select it.
- :guilabel:`Cost`: Enter the estimated cost of the service, if available. If the service is for a
  future repair, this field should be left blank. This field is updated as estimates are received,
  and again, when the final repair cost is known.
- :guilabel:`Vendor`: Using the drop-down menu, select the vendor who is performing the service. If the
  vendor has not already been entered in the system, :ref:`add and configure the vendor
  <fleet/new-vendor>`.
- :guilabel:`Vehicle`: Using the drop-down menu, select the vehicle that was serviced. When the
  vehicle is selected, the :guilabel:`Driver` field is populated, and the unit of measure for the
  :guilabel:`Odometer Value` field appears.

> - :guilabel:`Driver`: The vehicle's current driver populates this field when the :guilabel:`Vehicle`

Reviewer: I think you need passive "is populated" or "is automatically populated" because this sounds like a literal person is doing something. #avoidpronounlife

  is selected. If the driver needs to be changed, another driver can be selected using the drop-down menu.
- :guilabel:`Odometer Value`: Enter the odometer reading from when the service was done. The units of
  measure are either in kilometers (:guilabel:`km`) or miles (:guilabel:`mi`), depending on how the
  selected vehicle was configured.

  .. tip::
     To change from kilometers to miles, or vice versa, click the :icon:`oi-arrow-right`
     :guilabel:`(Internal Link)` icon to the right of the vehicle selected in the :guilabel:`Vehicle` field.

     Change the unit of measure, then navigate back to the service form, via the breadcrumb links.
     The unit of measure is then updated in the :guilabel:`Odometer Value` field.

- :guilabel:`NOTES`: Enter any notes for the repair at the bottom of the service form. For example, this
  can include estimate details or parts being replaced.

.. image:: service/new-service.png
   :alt: Enter the information for a new service. The required fields are Service Type and Vehicle.

.. _fleet/new-type:

Create service type
-------------------

Service types should be created from a :ref:`service form <fleet/service-form>`. No other way exists

> to access the list of service types.

Reviewer: I'm assuming this was a style guide conformity thing, but "No other way exists" is a cursed-sounding construction. [Shikata ga nai](https://en.wikipedia.org/wiki/Shikata_ga_nai).

Author: Yup! I believe this was an edit from someone else? I can't honestly remember, but it was super-important to mention (and kind of odd for Odoo to do it this way).

  :guilabel:`Vehicle` is selected. If the driver needs to be changed, another driver can be selected
  using the drop-down menu.
- :guilabel:`Odometer Value`: Enter the odometer reading from when the service was done. The units
  of measure are either in kilometers (:guilabel:`km`) or miles (:guilabel:`mi`), depending on how
  the selected vehicle was configured.

  .. tip::
     To change from kilometers to miles, or vice versa, click the :icon:`oi-arrow-right`
     :guilabel:`(Internal Link)` icon to the right of the vehicle selected in the :guilabel:`Vehicle`
     field.

     Change the unit of measure, then navigate back to the service form, via the breadcrumb links.
     The unit of measure is then updated in the :guilabel:`Odometer Value` field.

- :guilabel:`NOTES`: Enter any notes for the repair at the bottom of the service form. For example,
  this can include estimate details or parts being replaced.

.. image:: service/new-service.png
   :alt: Enter the information for a new service. The required fields are Service Type and Vehicle.

.. _fleet/new-type:

Create service type
-------------------

> Service types should be created from a :ref:`service form <fleet/service-form>`. No other way exists
> to access the list of service types.

Reviewer 2: Here's a more formal way to communicate the info with an emphasis on it to be only way (:
```suggestion
The **only** method to create service types is from a :ref:`service form <fleet/service-form>`.
```

Create vendor
-------------

> When a service is done for the first time, typically, the vendor is not in the database. It is best

Reviewer 2: ```suggestion
When a service is performed for the first time, typically, the vendor's record has not yet been added to the database. It is best
```
